.github/workflows/ci.yml

```diff
@@ -162,5 +162,16 @@ jobs:
           nim --version
           nimble --version
           nimble install -y --depsOnly
-          env TEST_LANG="c" nimble test
-          env TEST_LANG="cpp" nimble test
+          env TEST_LANG="c" NIMFLAGS="${NIMFLAGS} --gc:refc" nimble test
+          env TEST_LANG="cpp" NIMFLAGS="${NIMFLAGS} --gc:refc" nimble test
+          if [[ "${{ matrix.branch }}" == "devel" ]]; then
+            echo -e "\nTesting with '--gc:orc':\n"
```

Comment:
There is memory corruption going on in the new `Opt/Option` test cases when using `devel` with `--gc:orc`.
Switched to `--gc:refc` for now, and marked this in a way so that we will know when it works again (the `if` wrappers around the `--gc:orc` `nimble test` commands need to be removed).

Sadly, did not manage to provide minimal repro in reasonable time. Maybe something related to the `lent` results from `Option`/`Opt`. It affects both.